ibis/backends/mysql/__init__.py

```diff
@@ -474,7 +481,7 @@ def _in_memory_table_exists(self, name: str) -> bool:
                 cur.fetchall()
         except pymysql.err.ProgrammingError as e:
             err_code, _ = e.args
-            if err_code == ER.NO_SUCH_TABLE:
```

Comment:
Any reason to change the import here?

Reply:
This show up as a conflict when merging main because it was initially without the `ER.` on my end but this was used in other PR with that was merged added into main with `ER.`, happy to unify it with `ER.`